notification/search.go

```diff
@@ -217,35 +253,69 @@ func (s *Store) TimeSeries(ctx context.Context, opts TimeSeriesOpts) ([]TimeSeri
 	}
 	defer rows.Close()
 
-	counts := make(map[int]int)
+	buckets := make(map[bucketID]int) // bucketID{}: count
```

Comment:
```suggestion
	buckets := make(map[bucketID]int)
```

Reply:
the comment is to let anyone know what the `int` represents

Reply:
Well, it is a leftover note, but I thought it would be useful for others to see in the future, as `int` is ambiguous.